documentation/asciidoc/topics/attributes/community-attributes.adoc

```diff
@@ -2,6 +2,10 @@
 :ispn_operator: Infinispan Operator
 :infinispanversion: 9.4.0
 :k8s: Kubernetes
+:okd: OKD
+
+:operator_root: https://github.com/infinispan/infinispan-operator/
+:operator_raw: https://raw.githubusercontent.com/rigazilla/infinispan-operator/dg_startup/
```

Comment:
We should use inifnispan group instead rigazilla user right?

Reply:
infinispan is upstream but what about rhdg 7.3? should we tell customers to install the product resources from the community repo?